packages/core/src/execute.js

```diff
@@ -9,31 +9,27 @@ const createJSONtool = require('./tools/create-json-tool');
 const ensureArray = require('./tools/ensure-array');
 const injectInput = require('./http-middlewares/before/inject-input');
 const prepareRequest = require('./http-middlewares/before/prepare-request');
-const throwForStatus = require('./http-middlewares/after/throw-for-status');
 const ZapierPromise = require('./tools/promise');
 const { FORM_TYPE } = require('./tools/http');
 
 const constants = require('./constants');
 
 const executeHttpRequest = (input, options) => {
   options = _.extend({}, options, constants.REQUEST_OBJECT_SHORTHAND_OPTIONS);
-  return input.z
-    .request(options)
-    .then(throwForStatus)
```

Comment:
No need to append `throwForStatus` middleware for configured requests as all requests now default to it.

Reply:
when does executeHttpRequest actually get called? I wonder if this is only for a form mode app? Just wondering because I don't think I've seen this get called for code CLI apps or in code mode visual buildler apps?

Reply:
We call `executeHttpRequest` when a "request shorthand" is used. Namely, when you write something like this for a `perform` or auth `test`:

```
{
  method: 'POST',
  url: '...',
  body: {
    foo: '{{bundle.inputData.foo}}'
  }
}
```

This is also the underlying representation for visual builder's form mode.

Reply:
hmmm @FokkeZB sounds like this could also be used in a CLI app then, not just visual builder? Not sure if we considered that case.

Reply:
Yes, I never realized before, but preventing `throwForStatus` from throwing or handling unparsable content isn't something you can currently do (without the hacky overwriting of status and content) in CLI apps that have at least one shorthand/form-mode request either, as there as well we always run `throwForStatus` and `JSON.parse(response.content)`.

So this indeed will benefit CLI developers as well.